apps/web/src/server/applications/pages/case/documentation/applications-documentation.controller.js

```diff
@@ -10,9 +16,41 @@ import { getCaseDocumentationCategories } from '../../../lib/services/case.servi
  */
 export async function viewApplicationsCaseDocumentationCategories(request, response) {
 	const { caseId } = response.locals;
-	const documentationCategories = await getCaseDocumentationCategories(caseId);
-
+	const documentationCategories = await getCaseFolders(caseId);
 	const properties = { documentationCategories: sortBy(documentationCategories, ['displayOrder']) };
 
 	response.render(`applications/case/project-documentation`, properties);
 }
+
+/**
+ * View a folder, showing files in the folder, and listing subfolders
+ *
+ * @param {*} request
```

Comment:
I know this was in the previous PR, but I didn't see it.
Can you change it to the usual syntax:

`@type {import('@pins/express').RenderHandler<PROPERTIES_STYPE, {}, {}, {}, {}>}`